pkg/kubelet/cm/cgroup_manager_test.go

```diff
@@ -0,0 +1,94 @@
+// +build linux
+
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cm
+
+import (
+	"testing"
+
+	"github.com/opencontainers/runc/libcontainer/configs"
+)
+
+func NewMockLibcontainerManager(mockObject *MockLibcontainerManager) *libcontainerCgroupManager {
+	return &libcontainerCgroupManager{
+		fsCgroupManager: mockObject,
+	}
+}
+
+func TestLibcontainerCgroupManager_Create(t *testing.T) {
+	mockObject := &MockLibcontainerManager{}
+	libcontainerConfig := &configs.Config{
+		Cgroups: &configs.Cgroup{
+			Name:   "foo",
+			Parent: "/",
+			Resources: &configs.Resources{
+				Memory:    31457280,
+				CpuQuota:  1000,
+				CpuShares: 2,
+			},
+		},
+	}
+	//Setup expectations
+	mockObject.On("Set", libcontainerConfig).Return(nil)
+	mockObject.On("Apply", 0).Return(nil)
+	// Call function that uses the mockObject
+
```

Comment:
call me crazy, but I dont really get what code this is testing.  is this testing the mock?  why do we care about testing the mock so much versus the actual implementation?